src/chains/ethereum/options/src/chain-options.ts

```diff
@@ -151,6 +167,13 @@ export const ChainOptions: Definitions<ChainConfig> = {
     legacyName: "allowUnlimitedContractSize",
     cliType: "boolean"
   },
+  allowUnlimitedInitCodeSize: {
+    normalize,
+    cliDescription:
+      "Allows unlimited initcode (`transaction.data`) sizes while debugging. Setting this to `true` will cause ganache to behave differently than production environments.",
```

Comment:
I think the description suggests that this only works in debugging context - I understand the intention, and feel that we've given sufficient warning with "...behave differently...".

```suggestion
      "Allows unlimited initcode (`transaction.data`) sizes. Setting this to `true` will cause ganache to behave differently than production environments.",
```

Reply:
The description mirrors the description for `allowUnlimitedContractSize`. Should we update both?

Reply:
Sure!